hurricane/server/__init__.py

```diff
@@ -194,3 +202,11 @@ def check_db_and_migrations(webhook_url: str = None, loop: asyncio.unix_events.S
         StartupWebhook().run(
             url=webhook_url or None, error_trace=error_trace, close_loop=False, status=WebhookStatus.WARNING, loop=loop
         )
+
+
+def add_trailing_slash(options):
+    # adding optional / to the regular expression of probe handler
+    options["liveness_probe"] = options["liveness_probe"] + "/{0,1}"
```

Comment:
the regular expression for 
`options["liveness_probe"]` to  /healthz/ (`--liveness-probe=/healthz/`) would then be `/healthz//{0,1}`
Is that the desired result?

Reply:
good point, this is not a desired result. I have added a logic to better deal with slashes and to generalize to other edge cases i.e. whitespaces in the probe string